File: core-java-practice/gcr codebase/programming-elements/SumOfTwoNumbers.java
import java.util.Scanner;
// Importing the scanner class , for taking user input 

public class SumOfTwoNumbers {
    public static void main(String[] args) {
        // Create a Scanner object to read input
        Scanner sc = new Scanner(System.in);

        // Asking the user for two numbers
        System.out.print("Enter the first number: ");
        int n1 = sc.nextInt();

        System.out.print("Enter the second number: ");
        int n2 = sc.nextInt();

        // Calculate the sum
        int sum = n1 + n2;

        // Print the result
        System.out.println("The sum of " + n1 + " and " + n2 + " is: " + sum);

        // Close the scanner
        sc.close();
    }
}

File: core-c#-practice/gcr codebase/programming-elements/SumOfTwoNumbers.cs
using System;
class Sum_of_two
{
    public static void Main ()
    {
        Console.WriteLine ("Enter the num1 : "); // Asking the user for first no.
        int a = int.Parse ( Console.ReadLine () ) ; // Taking the value and converting it to integer using Parse method
        // Used Parse method ( static ) to convert string input to target type which is int here
        // Using int here because we are only dealing with whole numbers

        Console.WriteLine ("Enter the num2 : "); // Asking the user for second no.
        int b = int.Parse ( Console.ReadLine () ) ; // Taking the value and converting it to integer using Parse method
        // Used Parse method ( static ) to convert string input to target type which is int here
        // Using int here because we are only dealing with whole numbers

        int sum = a + b ; // Adding the two numbers
        Console.WriteLine ( " The sum of " + a + " and " + b + " is : " + sum ) ; // Showing the sum of two numbers
    }
}
